Tasks/powerShell/task.json

```diff
@@ -50,13 +64,25 @@
             "required":false,
             "helpMarkDown": "Current working directory when script is run.  Defaults to the folder where the script is located.",
             "groupName":"advanced"
+        },
+        {
+            "name": "script",
+            "type": "multiLine",
+            "label": "Script",
+            "defaultValue": "Write-Host 'Hello World'",
+            "required": true,
+            "helpMarkDown": "Write your scripts and have fun",
+            "visibleRule": "type = InlineScript",
+            "properties": {
+                "rows": 5
+            }
         }
     ],
     "instanceNameFormat": "Powershell: $(scriptName)",
     "execution": {
-        "PowerShellExe": {
-            "target": "$(scriptName)",
-            "argumentFormat": "$(arguments)",
+        "PowerShell": {
```

Comment:
I need a logic deciding between whether the task will execute the inline powershell or the powershell script depending upon the type passed. The Exe handler earlier was used only to execute the powershell script.